lib/ansible/modules/cloud/amazon/rds_snapshot_info.py

```diff
@@ -359,7 +360,7 @@ def standalone_snapshot_info(module, conn):
         if snapshot_type == 'public':
             params['IsPublic'] = True
         elif snapshot_type == 'shared':
-            params['IsShared'] = True
+            params['IncludeShared'] = True
```

Comment:
Hm I wonder if there was an API change? It looks like `params['IsPublic'] = True` has the same bug and should also be `params['IncludePublic'] = True`, would you be willing to update that as well?

Reply:
Oh, happy to change that, too, I wasn't aware!

Reply:
@jillr Changed, and added a test!